src/dataflow/src/render/reduce.rs

```diff
@@ -213,24 +248,42 @@ where
                             *expected_group_size,
                         );
                         to_collect.push(collection.as_collection(move |key, val| {
-                            (key.clone(), (Some(index), val.clone()))
+                            (
+                                key.clone(),
+                                (ReductionType::Other, Some(index), val.clone()),
+                            )
                         }));
                     }
                     let is_accumulable = aggregates
                         .iter()
                         .map(|a| accumulable_hierarchical(&a.func).0)
                         .collect::<Vec<_>>();
+                    let is_min_max = aggregates
+                        .iter()
+                        .map(|a| is_min_or_max(&a.func) && !monotonic)
+                        .collect::<Vec<_>>();
+
                     differential_dataflow::collection::concatenate(scope, to_collect)
                         .reduce_abelian::<_, OrdValSpine<_, _, _, _>>("ReduceCollation", {
                             let mut row_packer = RowPacker::new();
                             move |key, mut input, output| {
-                                // The inputs are pairs of an optional index and row to decode.
+                                // The inputs are triples of a reduction type, an optional index and row to decode.
 
                                 // There can be at most one `None` index, and it indicates the accumulable aggregates.
```

Comment:
This comment is now wrong, and probably misleading for a new reader.

Reply:
yes thank you!